website/src/controllers/user.js

```diff
@@ -117,12 +117,26 @@ api.score = function(req, res, next) {
     }, saved.toJSON().stats));
 
     // Webhooks
+
+    // Select character data to send
+    var userData = _.pick(user.toJSON(), ['_id', '_tmp', 'stats']); // user.toJSON to copy-by-value
```

Comment:
user.toJSON() is heavy :( since we're not saving the user anywhere we could just mutate the original object. Though I'm not sure if mongoose is going to allow you to set new properties that are extraneous to the model